xline-test-utils/src/lib.rs

```diff
@@ -189,9 +195,16 @@ impl Cluster {
 
 impl Drop for Cluster {
     fn drop(&mut self) {
-        for path in self.paths.values() {
-            let _ignore = std::fs::remove_dir_all(path);
-        }
+        block_in_place(move || {
```

Comment:
Yes, it will, all the tests that use this need to run in a multi-thread runtime

Reply:
> Could we call `async fn stop()` maunually instead of putting it into `Drop`?

This is the original way, which is not as simple as the present one, and easily ignored.